custom-vu/src/main/kotlin/jces1209/vu/page/dashboard/DashboardPage.kt

```diff
@@ -0,0 +1,84 @@
+package jces1209.vu.page.dashboard
+
+import com.atlassian.performance.tools.jiraactions.api.page.wait
+import org.openqa.selenium.By
+import org.openqa.selenium.JavascriptExecutor
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.support.ui.ExpectedConditions
+import org.openqa.selenium.support.ui.WebDriverWait
+import java.net.URI
+import java.time.Duration
+
+
+abstract class DashboardPage(
+    protected val driver: WebDriver,
+    protected val uri: URI
+) {
+
+
+    fun waitForDashboards() {
+        driver.wait(
+            condition = ExpectedConditions.visibilityOfElementLocated(
+                By.xpath("//a[contains(@href,'/secure/Dashboard.jspa?selectPageId')]")),
+            timeout = Duration.ofSeconds(50)
+        )
+    }
+
+    fun createDashboard() {
+        driver.findElement(
+            By.xpath("(//button[@data-test-id='directory.dashboards-v2.create-button'])")).click()
```

Comment:
I can't see such locators on DC page. Is it really working?
: /secure/ConfigurePortalPages.jspa

Reply:
fixed